compiler/include/ForallStmt.h

```diff
@@ -55,30 +93,39 @@ class ForallStmt : public Stmt
   Expr* firstIteratedExpr() const;
   int   numIteratedExprs() const;
   bool isIteratedExpression(Expr* expr);
-  int  reduceIntentIdx(Symbol* var); // todo: replace with LoopIntentVars
+  int  reduceIntentIdx(Symbol* var); // todo: replace with ForallIntents
+  int           numFIntents() const;
+  ForallIntent* getFIntent(int index) const; //todo replace with LoopIntentVars
 
 private:
   bool           fZippered;
   AList          fIterVars;
   AList          fIterExprs;
-  AList          fIntentVars;  // may be empty
+  AList          fIntentVars;  // may be empty; elems are ForallIntent
+  AList          fFIntents;    // todo: replace with intentVariables
```

Comment:
Can you explain what the difference is between fIntentVars  and fFIntents? The comment says fIntentVars contains ForallIntent elements. Are the comments just reversed?